src/seahawk/seahawk_deck/thrustverify.py

```diff
@@ -0,0 +1,79 @@
+'''
+thrustverify.py
+
+Calculate predicted thrust and angular vector based on motor status and output it on /drive/predict
+
+Copyright (C) 2022-2023 Cabrillo Robotics Club
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Cabrillo Robotics Club
+6500 Soquel Drive Aptos, CA 95003
+cabrillorobotics@gmail.com
+'''
+import sys 
+
+import rclpy
+
+from rclpy.node import Node 
+
+from geometry_msgs.msg import Twist
+from std_msgs.msg import Float32MultiArray
+
+import numpy as np
+
+class ThrustVerify(Node):
+    """Caculates expected input, based on current motor status"""
+
+    def __init__(self):
+        super().__init__('thrust')
+
+        self.subscription = self.create_subscription(Float32MultiArray, 'drive/motors', self.motor_math, 10)
+        self.vector_predict = self.create_publisher(Float32MultiArray, 'drive/predict', 10)
+
+    def motor_math(self, msg):
+        # motor 0-7 within list index
+        motor_msg = msg
+
+        config = [
+            [0, 0, 0, 0, 0.7071, 0.7071, -0.7071, -0.7071], 
+            [0, 0, 0, 0, -0.7071, 0.7071, -0.7071, 0.7071], 
+            [1, 1, 1, 1, 0, 0, 0, 0],
+            [-.19, -.19, .19, .19, -.105, -.105, .105, .105],
+            [-.12, .12, -.12, .12, -.15, .15, -.15, .15],
+            [-.047,-.047,-.047, -.047, 0.038, 0.038, 0.038, 0.038]
+        ]
```

Comment:
Can you add a brief code comment describing what exactly this is, and how we got it? We can save specifics for markdown documentation (should probably write that too!)

Reply:
These latest changes (757736b, 8bce7db), while they do help, don't particularly resolve describing what `config` is exactly. It's good you are describing the components of it, however.